optuna/visualization/_contour.py

```diff
@@ -61,6 +62,11 @@ def objective(trial):
         A :class:`plotly.graph_objs.Figure` object.
     """
 
+    if len(study.trials) > 1000:
+        warnings.warn(
+            "Plotting {} trials, which may cause unstable behavior"
+            " of the visualization feature with too many trials."
```

Comment:
Should the number for actual trials be included in the string considering there's a placeholder?

Reply:
@hvy  Hi I changed it to actual number of 1000 and also the number are now based on completed trials